### DISCUSSION TITLE
CH560 Digital works

### DISCUSSION THREAD

**Discussion by Thazulk** [Nortank12/deepcool-digital-linux#8] [General]
Good afternoon!

Works on CH560 with Ryzen 7 7800X3D on arch linux.

Had to tweak it a little because by default it didn't get any cpu temp readings. 
This solved my issue:

https://wiki.archlinux.org/title/Lm_sensors#K10Temp_module

Ty, Cheers!

**Comment by Nortank12** [comment]
Thanks for the feedback! I've marked the CH560 Digital as tested.
